examples/cdc-advanced-service/cdc_advanced_service.metatags

```diff
@@ -0,0 +1,2 @@
+description: Thisn BBE demonstrates how to use a CDC (Change Data Capture) service in Ballerina to capture and process real-time changes from a MySQL database.
```

Comment:
```suggestion
description: This BBE demonstrates how to use a CDC (Change Data Capture) service in Ballerina to capture and process real-time changes from a MySQL database.
```

Reply:
Fixed in [85aa9b7](https://github.com/ballerina-platform/ballerina-distribution/pull/6074/commits/85aa9b7e654a44c01e262224a7dc88f9da8a7836)